core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -125,10 +119,9 @@ class UserServiceSpec
 
       val userService = getUserService(dataSource, servicePerimeterService = mockServicePerimeterService)
 
-      val actual =
+      val actual: Unit =
         userService.addProjectToServicePerimeter(defaultServicePerimeterName, project.projectName).futureValue
-      val expected = ()
-      actual shouldEqual expected
+      actual shouldEqual ()
```

Comment:
This doesn't seem like a meaningful assertion; if the future did anything else the test would fail anyway wouldn't it?

Reply:
That's correct. I think I auto-resolved that line from an IntelliJ inspection..